metadata.rb

```diff
@@ -1,8 +1,15 @@
-name             'timesync'
-maintainer       'Oregon State University'
+name 'timesync'
```

Comment:
Oh, looks like it's already in there. I guess I must've been looking at my editor's rubocop when I changed that.